source/documentation/tools/jupyterlab/package-management.md

```diff
@@ -0,0 +1,87 @@
+# Python Package Management
+
+## venv and pip
+
+[pip](https://pip.pypa.io/en/stable/) is a terminal command used to install and upgrade Python packages.
+
+[PyPI](https://pypi.org) is the main Python package repository. It's 'official', but that doesn't mean a lot - like most of these open source package repositories, a poor quality or even malicious package can easily be uploaded there, so do your diligence when picking them.
+
+A [Python virtual environment](https://docs.python.org/3/tutorial/venv.html) (or `venv`, for short) is a directory you can install a particular python executable and python packages into, away from your machine's default ones. Typically each project/repo you work on should have a different venv, and then you never have to deal with conflicting requirements between projects. When you 'activate' a particular venv, then when you run `python` or `pip`, then it will work with that venv's python executable and python packages.
+
+## Basic usage
```

Comment:
Not necessarily something for this pull request to solve, but there's some confusing repetition between the 'JupyterLab' page, which has a section on 'Using a virtual environment in Jupyter', and this section, which is part of a page called 'Python package management'. Currently the package management page is a directly child of the JupyterLab page.

I suspect the best option is to delete the virtual environment material from the JupyterLab page, as the stuff here on the package management page is more visible in the navigation.